susemanager/bin/pg-migrate-12-to-13.sh

```diff
@@ -92,8 +92,8 @@ else
     echo "`date +"%H:%M:%S"`   Initialization of new postgresql $NEW_VERSION database failed!"
     echo "`date +"%H:%M:%S"`   Trying to restore previous state..."
     mv /var/lib/pgsql/data /var/lib/pgsql/data-new-failed
-    mv /var/lib/pgsql/data-pg10 /var/lib/pgsql/data
-    /usr/sbin/update-alternatives --set postgresql /usr/lib/postgresql10
+    mv /var/lib/pgsql/data-$OLD_VERSION /var/lib/pgsql/data
```

Comment:
```suggestion
    mv /var/lib/pgsql/data-pg$OLD_VERSION /var/lib/pgsql/data
```

Reply:
I'd not change this, to keep it as we have for the other packages. We don't use `data-pg` anywhere else, but up to Jordi.

Reply:
On a contrary. This is what we use everywhere. `OLD_VERSION` is just a number.

Reply:
See https://github.com/uyuni-project/uyuni/blob/master/susemanager/bin/pg-migrate-12-to-13.sh#L79

Reply:
@aaannz Thanks! That was a mistake from my side. Fixed now.

Reply:
Right,. I basically blind :-D